class-amp-content.php

```diff
@@ -32,12 +32,15 @@ public function transform() {
 
 		$content = AMP_Sanitizer::strip( $content );
 
+		$img_converter = new AMP_Img_Converter( $content );
 		// Convert HTML to AMP
 		// see https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#html-tags)
-		$content = $this->convert( new AMP_Img_Converter( $content ), array(
+		$content = $this->convert( $img_converter, array(
 			'layout' => 'responsive',
 		) );
 
+		$this->add_scripts( $img_converter->get_scripts() );
```

Comment:
The `convert` method calls `add_scripts` for us so we shouldn't need to call it here.